modules/server_statisticts/voice_statistics.py

```diff
@@ -0,0 +1,43 @@
+import discord
+from discord import Embed
+from discord.ext import commands
+from discord.ui import View
+from discord.utils import get
+
+from utils.command import command
+from utils.database import Database
+from utils.errors import DiscordException
+
+
+class VoiceStatistics(commands.GroupCog, group_name='voicestats'):
+    def __init__(self, bot):
+        self.bot = bot
+
+    def get_user_name(self, id):
+        user = get(self.bot.get_all_members(), id=id)
+        if user is None:
+            return "User not found"
+        return user.name
+
+    async def generate_top10(self, ctx, list):
+        top = ""
+        for record in list:
+            top += self.get_user_name(record[0]) + " : " + str(round(record[1])) + " s\n"
+
+        await ctx._send(
+            content="",
+            embed=Embed(title="TOP 10 VOICE USERS",
+                        description=top,
+                        color=discord.Color.blue()),
+            view=View(timeout=None)
+        )
+
+    @command(name="general", description="Get top 10 active users on voice channels", is_hidden=False)
+    async def top10_voice_channels(self, ctx):
```

Comment:
This naming is miss leading. ( for me it mean: "top ten most popular voise channels" not "top ten most active people's on all voice channels")